src/main/java/org/example/models/Diagnosis.java

```diff
@@ -0,0 +1,85 @@
+package org.example.models;
+import jakarta.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "diagnosis")
+public class Diagnosis {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @ManyToOne
+    @JoinColumn(name = "doc_id", nullable = false)
+    private Doctor doctor;
+
+    @Column(name = "symptoms", nullable = false)
+    private String symptoms;
```

Comment:
To make it more real-life, use diagnosis code to refer to some diseases. 
https://icd.who.int/browse10/2019/en 
So we'd have somethis like 

@Column(name = "diagnosis_code", nullable = false)
private String diagnosisCode;

Reply:
Later we'd need to add some validation that code starts with ICD-9/10 (for ex.)

Reply:
Done, validation will be added later